# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

PickOrPlaceActionGoal action_goal
PickOrPlaceActionResult action_result
PickOrPlaceActionFeedback action_feedback
